My feature

diegoh40 opened the pull request.

diegoh40 committed:
add index.html file

diegoh40 committed:
add README.md file

diegoh40 committed:
add styles.css file

Donard97 reviewed (changes requested):
## Hi @diegoh40 , 👋  

_You did an amazing job so far! 👏 💯_ 

**There are some issues that you still need to work on to go to the next project but you are almost there! 💪** 

![hgiphy](https://user-images.githubusercontent.com/74506933/164352058-c77059ea-859e-4262-bd8b-00d3d5f6d35e.gif)


## Highlights ✅ 

✔️ I can see Hello Microverse
✔️ There are a few meaningful commits
✔️ Gitflow is used correctly

## Required Changes ♻️

- [ ] Kindly install linters following [linters configuration repository](https://github.com/microverseinc/linters-config/tree/master/html-css)
- [ ] Please add a more descriptive title and add a description for PULL REQUEST for example you have to tell use what you did in this branch.
- [ ] Kindly add .gitignore file and list there test.md ( because it was part of the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) to add a test.md file with the  text "This file should be ignored by git") and node_modules because they can be added when you will install linters
- [ ] I suggest you in the feature when you create new branches name them in base of what you are doing in this branch. For example in this case you name it my-feature but it would be perfect if you would name it let's say  add-hello-microverse

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

diegoh40 committed:
add .github/workflows/linters.yml file

diegoh40 committed:
add .gitignore file

diegoh40 commented:
I have added .ignore and added .github/workflow to this project and realized that I need to customize the branch names in my future new branches, as you mention in your comments thanks.

nicupop729 reviewed (commented):
Hi @diegoh40, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<img src="https://media3.giphy.com/media/cKKXNlTYino7hWNXwl/giphy.gif"/>

You have submitted a project where the branch is already merged with the master.
**Kindly, redo your project. In the description of that pull request, from the future project, please mention what happened**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
